src/encoded/search_views.py

```diff
@@ -218,6 +219,41 @@ def matrix(context, request):
     return fr.render()
 
 
+@view_config(route_name='chip-seq-matrix', request_method='GET', permission='search')
+def chip_seq_matrix(context, request):
+    fr = FieldedResponse(
+        _meta={
+            'params_parser': ParamsParser(request)
+        },
+        response_fields=[
+            TitleResponseField(
+                title='ChIP-seq Matrix'
+            ),
+            TypeResponseField(
+                at_type=['ChipSeqMatrix']
+            ),
+            IDResponseField(),
+            SearchBaseResponseField(),
+            ContextResponseField(),
+            BasicMatrixWithFacetsResponseField(
+                default_item_types=DEFAULT_ITEM_TYPES,
+                matrix_definition_name='chip_seq_matrix',
+                facets=[
```

Comment:
I don't think these facets are actually used?

Reply:
they are used.

Reply:
Where?

Reply:
I missed read what you wrote. Yeah, it was facets are not needed as they were used in an early iteration that has since changed. BasicMatrixWithFacetsResponseField is needed though.